src/frontend/js/components/AshleyEditor/index.tsx

```diff
@@ -86,19 +85,17 @@ const AshleyEditor = (props: MyEditorProps) => {
   };
 
   return (
-    <div>
-      <div
-        className="ashley-editor-widget"
-        ref={editorContainerRef}
-        onClick={focusEditor}
-        style={
-          toolbarRef.current
-            ? {
-                top: `${toolbarRef.current.offsetHeight}px`,
-              }
-            : {}
-        }
-      >
+    <div
+      className="ashley-editor-wrapper"
+      style={
```

Comment:
By using `flex-direction: column-reverse`, you do not need anymore to add a padding to the wrapper.

Reply:
That's right. :pray: